src/type-hierarchy.ts

```diff
@@ -0,0 +1,213 @@
+import * as path from 'path';
```

Comment:
given that type-hierarchy is not in LSP yet, would be nice to have a high-level comment on the top of this file, e.g.  give a reference link of the type-hierarchy proposal.